README.md

```diff
@@ -1,10 +1,416 @@
+   the implementation can be found
+   [in the sources](src/core/tracking/skb_tracking.rs).
+
+2. A collector, `skb-tracking`, retrieves the core tracking information (unique
+   identifier and socket buffer address) and reports it in the event. Without
+   enabling this collector, skb tracking information won't be reported and can't
+   be used at post-processing time.
+
+3. The `ovs` collector tracks packets in upcalls so we can follow a packet
+   being sent to the OpenVSwitch user-space daemon, even if it is re-injected
+   later on.
+
+### Simple copy/paste examples
+
+Drop monitor-like, listing packets being dropped by the kernel with an
+associated stack trace and drop reason (in the example below, no socket was
+found).
+
+```
+$ retis -p drop collect
+00:42:00 [INFO] 4 probe(s) loaded
+
+28917566981729 [tp] skb:kfree_skb drop (NO_SOCKET)
+    bpf_prog_88089ccd9794be3a_sd_devices+0x247b3
+    bpf_prog_88089ccd9794be3a_sd_devices+0x247b3
+    bpf_trace_run3+0x52
+    kfree_skb_reason+0x8d
+    tcp_v6_rcv+0x77
+    ip6_protocol_deliver_rcu+0x6b
+    ip6_input_finish+0x43
+    __netif_receive_skb_one_core+0x62
+    process_backlog+0x85
+    __napi_poll+0x28
+    net_rx_action+0x29f
+    __do_softirq+0xd1
+    do_softirq.part.0+0x5f
+    __local_bh_enable_ip+0x68
+    ip6_finish_output2+0x1ec
+    ip6_finish_output+0x15d
+    ip6_xmit+0x2c0
+    inet6_csk_xmit+0xe9
+    __tcp_transmit_skb+0x537
+    tcp_connect+0xaf9
+    tcp_v6_connect+0x512
+    __inet_stream_connect+0x103
+    inet_stream_connect+0x3a
+    __sys_connect+0xa8
+    __x64_sys_connect+0x18
+    do_syscall_64+0x5d
+    entry_SYSCALL_64_after_hwframe+0x72
+  ::1.52952 > ::1.80 len 40 proto TCP (6) flags [S] seq 3808600276 win 65476
+...
+```
+
+Monitoring packets dropped by netfilter, the exact nft rule can be retrieved
+using `nft -a list table ...`.
+
+```
+$ retis -p nft-drop collect --allow-system-changes
+00:42:00 [INFO] 4 probe(s) loaded
+
+54645161861 [k] __nft_trace_packet
+  172.16.42.1.40238 > 172.16.42.2.2222 len 60 proto TCP (6) flags [S] seq 133522511 win 64240
+  table firewalld (1) chain filter_IN_FedoraServer (202) handle 215 drop
+...
+$ nft -a list table inet firewalld
+...
+	chain filter_IN_FedoraServer { # handle 202
+...
+		jump filter_INPUT_POLICIES_post # handle 214
+		meta l4proto { icmp, ipv6-icmp } accept # handle 273
+		reject with icmpx admin-prohibited # handle 215         <- This one
+	}
+...
+```
+
+## Limitations
+
+Known and current limitations:
+
+- By default Retis does not modify the system (e.g. load kernel modules, change
+  the configuration, add a firewalling rule). This is done on purpose but might
+  mean some prerequisites will be missing if not added manually. The only
+  example for now is the `nft` module that requires a specific nft rule to be
+  inserted. If that rule is not there, no nft event will be reported. To allow
+  Retis to modify the system, use the `--allow-system-changes` option when
```

Comment:
given that specific tables get added (with specific names), should we mention the names of the table and chain?
If not really needed, we might want to say something like (or equivalent):
"see --allow-system-changes help for further details about the changes applied to the system"